Jenkinsfile

```diff
@@ -151,24 +152,29 @@ def buildDockerEnv(name) {
     return docker.image(name)
 }
 
-def doBuildInDocker(String buildType) {
+def doBuildInDocker(String buildType, String sanitizeMode) {
```

Comment:
If you used a default value for the parameter you could avoid passing an empty string when calling this function:

```
def doBuildInDocker(String buildType, String sanitizeMode='') {
```